expat/coverage.sh

```diff
@@ -38,8 +38,10 @@ _get_coverage_dir() {
 _configure() {
     local configure_args=()
 
+    configure_args+=( CPPFLAGS='-DXML_ATTR_INFO' )
+
     ${unicode_enabled} \
-            && configure_args+=( CPPFLAGS='-DXML_UNICODE -DXML_UNICODE_WCHAR_T' )
+            && configure_args+=( CPPFLAGS+='-DXML_UNICODE -DXML_UNICODE_WCHAR_T' )
```

Comment:
That second `+=` looks wrong to me. Does configure support that?

Also gave it a try:
```console
# ./configure CPPFLAGS=X CPPFLAGS+=Y
configure: error: invalid variable name: `CPPFLAGS+'
```

Reply:
Updated review comments and uploaded the change